### PR TITLE
flesh out abstract (exactly 250 words)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: isc24/CernVM-FS/abstract.tex [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+
+CernVM-FS is a compelling solution for Continuous Delivery of software in HPC environments, since it has been proven to scale to billions of files and tens of thousands of clients. Once a software installation has been ingested in a CernVM-FS repository on a centrally managed server, it is
+automatically synchronized to a network of mirror servers and associated proxy servers.
+After a CernVM-FS repository has been made accessible on a system, no subsequent action must be taken by system administrators or end users to obtain updates to the contents of the repository, since that is done fully automatically by the client component of CernVM-FS via the network of mirror and proxy servers.
+
+This tutorial will introduce CernVM-FS through hands-on exercises and demos, and will cover specific aspects that are relevant to using it in an HPC context, including HPC-specific configuration, troubleshooting, performance, and containers. The European Environment for Scientific Software
+Installations (EESSI) will serve as example CernVM-FS repository.
```
Comment (ocaisa) [inline isc24/CernVM-FS/abstract.tex]:
```suggestion
Installations (EESSI) will serve as an example CernVM-FS repository.
```

Comment (ocaisa) [reply isc24/CernVM-FS/abstract.tex]: Eek 251!

File: isc24/CernVM-FS/abstract.tex [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+
+CernVM-FS is a compelling solution for Continuous Delivery of software in HPC environments, since it has been proven to scale to billions of files and tens of thousands of clients. Once a software installation has been ingested in a CernVM-FS repository on a centrally managed server, it is
+automatically synchronized to a network of mirror servers and associated proxy servers.
+After a CernVM-FS repository has been made accessible on a system, no subsequent action must be taken by system administrators or end users to obtain updates to the contents of the repository, since that is done fully automatically by the client component of CernVM-FS via the network of mirror and proxy servers.
+
+This tutorial will introduce CernVM-FS through hands-on exercises and demos, and will cover specific aspects that are relevant to using it in an HPC context, including HPC-specific configuration, troubleshooting, performance, and containers. The European Environment for Scientific Software
```
Comment (ocaisa) [inline isc24/CernVM-FS/abstract.tex]:
```suggestion
This tutorial will introduce CernVM-FS through hands-on exercises and demos, and cover specific aspects that are relevant to using it in an HPC context, including HPC-specific configuration, troubleshooting, performance, and containers. The European Environment for Scientific Software
```
Problem solved!

File: isc24/CernVM-FS/abstract.tex (hunk 1 above)
Comment (boegel) [reply isc24/CernVM-FS/abstract.tex]: I think it works without the `an` just fine?

**Review by ocaisa — APPROVED** [review]

**Pull request merged**